share_app/src/main/java/org/odk/share/activities/MainActivity.java

```diff
@@ -58,13 +61,15 @@ protected void onCreate(Bundle savedInstanceState) {
         LinearLayoutManager llm = new LinearLayoutManager(this);
         llm.setOrientation(LinearLayoutManager.VERTICAL);
         recyclerView.setLayoutManager(llm);
+        setupAdapter();
+        getSupportLoaderManager().initLoader(FORM_LOADER, null, this);
     }
 
     @Override
     protected void onResume() {
+        updateAdapter();
```

Comment:
Is there a particular reason as to why `super.onResume()` is not called the first statement?